src/cascadia/TerminalSettingsModel/FileUtils.h

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.

```

Comment:
I'm a bit confused by our header styles: til uses this brief format, while some files uses the verbose one. Can I use the brief one in new code? Or should it always be the verbose one outside of til?
But I'm also not sure if anyone reads the verbose variant to be honest...

Reply:
So, generally, we should put the verbose one in the header, but the brief one in the cpp.

For files/classes that are pretty self-explanatory, the brief one is fine imo. Like, this one is probably fine because "FileUtils" is pretty clear that this is just a bunch of useful functions when working with files.

For the other one (ApplicationState), it's more useful because...
1. it's more complicated of a concept. What is this class used for?
2. if we recognize the name, it's easier to reach out to them and find them (easier than a git blame, but not by much)